src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final double MAX_SIDE = 10;
+    private static final int RADIUS = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private final Random random = new Random();
+    private final ColorSupplier color = new ColorSupplier();
+    private final double firstSide = random.nextDouble() * MAX_SIDE;
+    private final double secondSide = random.nextDouble() * MAX_SIDE;
+    private final double thirdSide = random.nextDouble() * MAX_SIDE;
```

Comment:
these variables should not be class-level, because all generated figures will have the same parameters. generate sides for each figure instead